$NetBSD: patch-ah,v 1.1 2007/01/21 04:02:01 rillig Exp $

sunpro does not like non-static inline functions that refer to static
variables.

--- buffer.c.orig	1997-08-18 17:54:20.000000000 +0200
+++ buffer.c	2007-01-21 04:44:48.242585932 +0100
@@ -46,7 +46,7 @@ static int control_fd;
 /* This is a separate (but inlined) function to make it easier to implement */
 /* a threaded buffer */
 
-inline void
+static inline void
 audioBufferWrite(char *buf,int bytes)
 {
         write(buffer_fd, buf, bytes);
